src/quo/components/buttons/wallet_ctas/style.cljs

```diff
@@ -1,13 +1,9 @@
 (ns quo.components.buttons.wallet-ctas.style)
 
-(def container
-  {:padding-top        24
-   :padding-bottom     12
-   :padding-horizontal 20
-   :flex-direction     :row
-   :justify-content    :center
-   :flex               1
-   :max-height         106})
+(def inner-container
```

Comment:
moved this to the page using and passed via container-style - (also updated preview screen)